test/unit-tests/symbol/IdGenerator.spec.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2021 SYMBOL
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { expect } from 'chai';
+import { SymbolIdGenerator } from '../../../src/core/symbol/SymbolIdGenerator';
+
+describe('Symbol IdGenerator - TestVector', () => {
+    it('Can generate a random nonce', () => {
+        const nonce = SymbolIdGenerator.generateRandomMosaicNonce();
+        expect(nonce).not.be.be.undefined;
+        expect(nonce.length).to.be.equal(4);
+    });
+
+    it('Can generate namespaceId', () => {
+        const namespaceId = SymbolIdGenerator.generateNamespaceId('nem');
+        expect(namespaceId).not.be.be.undefined;
+        expect(namespaceId.toString(16).toUpperCase()).to.be.equal('84B3552D375FFA4B');
+    });
+
+    it('Can generate namespaceId', () => {
+        const namespaceId = SymbolIdGenerator.generateNamespaceId('xem');
+        expect(namespaceId).not.be.be.undefined;
+        expect(namespaceId.toString(16).toUpperCase()).to.be.equal('A60FC7C97DB58AFC');
+    });
```

Comment:
notice these tests have the same name. depending on the test running, only one might be getting run.

you could also refactor with a helper function since the only variance is the input and output